model/task_lifecycle.go

```diff
@@ -1198,8 +1201,74 @@ func UpdatePatchStatus(p *patch.Patch, versionStatus string) error {
 		if err = p.UpdateStatus(patchStatus); err != nil {
 			return errors.Wrapf(err, "updating patch '%s' with status '%s'", p.Id.Hex(), patchStatus)
 		}
+		if p.IsGithubPRPatch() {
+			if err = SendVersionStatusToGithub(p.Id.Hex(), p.GithubPatchData.BaseOwner, p.GithubPatchData.BaseRepo, p.GithubPatchData.HeadHash, "patch status change", "pr-task-reset"); err != nil {
+				return errors.Wrapf(err, "sending patch '%s' status to Github", p.Id.Hex())
+			}
+		}
+	}
+
+	return nil
+}
+
+func SendVersionStatusToGithub(id, owner, repo, ref, desc, job string) error {
+	catcher := grip.NewBasicCatcher()
```

Comment:
there's no point in this catcher since we always just return if there's an error